tests/php/Forms/CurrencyFieldTest.php

```diff
@@ -302,11 +302,14 @@ public function testInvalidCurrencySymbol()
         $field = new CurrencyField('Test', '', '$5.00');
         $validator = new RequiredFields();
 
-        DBCurrency::config()->update('currency_symbol', '€');
+        DBCurrency::config()->set('currency_symbol', '€');
         $result = $field->validate($validator);
 
         $this->assertFalse($result, 'Validation should fail since wrong currency was used');
         $this->assertFalse($validator->getResult()->isValid(), 'Validator should receive failed state');
-        $this->assertStringContainsString('Please enter a valid currency', $validator->getResult()->serialize());
+        $this->assertStringContainsString(
+            'Please enter a valid currency',
+            json_encode($validator->getResult()->__serialize())
```

Comment:
Why we encode `$validator->getResult()->__serialize()`?

Reply:
Because the new __serialize() methods returns an [array](https://www.php.net/manual/en/language.oop5.magic.php#object.serialize).  This unit test uses `assertStringContainsString() `

The previous serialize() method returns a [string](https://www.php.net/manual/en/function.serialize.php)